scheduler/scheduler.go

```diff
@@ -207,22 +206,26 @@ func (s *Scheduler) Sync(ctx context.Context, client schema.ClientMeta, tables s
 			panic(fmt.Errorf("unknown scheduler %s", s.strategy.String()))
 		}
 	}()
+
+	const (
+		rows    = 50
```

Comment:
Can we make these configurable by the source plugin?
I would also default the rows to 1 (e.g. no batching), so we can roll this out gradually while we test additional sources.
We tested AWS so far, so we could make 50 the default for AWS

Reply:
f01a1ff199b0d68a7136d5d14938083deb906e08